fred2/sexp_tree.h

```diff
@@ -295,7 +295,8 @@ class sexp_tree : public CTreeCtrl
 	sexp_list_item *get_listing_opf_wing_flags();
 	sexp_list_item *get_listing_opf_team_colors();
 	sexp_list_item *get_listing_opf_nebula_patterns();
-	sexp_list_item* get_listing_opf_asteroid_debris();
+	sexp_list_item *get_listing_opf_asteroid_debris();
+	sexp_list_item *get_listing_opf_motion_debris();
```

Comment:
The changes in sexp_tree.h and sexp_tree.cpp should be replicated in their qtfred counterparts.

Reply:
done